crates/sui-rpc-api/src/client/mod.rs

```diff
@@ -150,7 +154,7 @@ impl Client {
 
         let (metadata, response, _extentions) = self
             .raw_client()
-            .max_decoding_message_size(64 * 1024 * 1024)
+            .max_decoding_message_size(128 * 1024 * 1024)
```

Comment:
One of the checkpoint in our test environment exceeded 64Mb, I bumped the max decoding message size to avoid the error, @bmwill do you know what is the correct value to set here?

Reply:
i think we really don't have a choice but to increase this...big checkpoints can be big it seems